src/Domain/Block.cpp

```diff
@@ -44,8 +44,9 @@ std::ostream& operator<<(std::ostream& os,
 template <size_t VolumeDim, typename TargetFrame>
 bool operator==(const Block<VolumeDim, TargetFrame>& lhs,
                 const Block<VolumeDim, TargetFrame>& rhs) noexcept {
-  return (lhs.id() == rhs.id() and lhs.neighbors() == rhs.neighbors() and
-          lhs.external_boundaries() == rhs.external_boundaries());
+  return lhs.id() == rhs.id() and lhs.neighbors() == rhs.neighbors() and
+         lhs.external_boundaries() == rhs.external_boundaries() and
+         lhs.coordinate_map() == rhs.coordinate_map();
```

Comment:
Again, I'm pretty sure we decided we didn't want this functionality. Was that changed?

Reply:
The operator should either be removed or it should be work correctly